activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -124,6 +124,10 @@ def supports_partial_index?
         true
       end
 
+      def supports_partitioned_indexes?
+        false
+      end
```

Comment:
This shouldn't be necessary, since it's identical to the version in AbstractAdapter.